espnet2/enh/encoder/stft_encoder.py

```diff
@@ -120,7 +120,6 @@ def _reconfig_for_fs(self, fs):
         Args:
             fs (int): new sampling rate
         """  # noqa: H405
-        assert fs % self.default_fs == 0 or self.default_fs % fs == 0
```

Comment:
Why this assertion was removed?

Reply:
This is to allow the process of speech with a sampling rate that is fractional of the encoder's default sampling rate. It doesn't have to be exactly 1/n or a multiple, so I removed this line.